test/test_vid.py

```
@@ -24,6 +25,17 @@ def test_generate_frames_cv2(video_path: Path):
         assert array.shape == (2160, 3840, 3)  # 4k colour
 
 
+def test_generate_frames_cv2_rtsp(rtsp_server: str):
```

Ruff (0.11.9)

Missing return type annotation for public function `test_generate_frames_cv2_rtsp`

Add return type annotation: `None`

(ANN201)